components/country/websites-charts.js

```diff
@@ -139,16 +139,19 @@ class TestsByCategoryInNetwork extends React.Component {
           </FormattedMessage>
         */}
         {/* URL-wise barcharts Start */}
-        {fetching && renderLoader()}
-        {(!fetching && testedUrls) &&
+        <Box style={{'min-height': '500px'}}>
```

Comment:
It maybe doesn't actually have to be `Box` it can just be a `div`

Reply:
Does it fix the issue though? I still see jitters as the page renders more stuff. Moreover, it creates another problem to solve on mobile and other layouts, which is why I chose not to go that route. It seems to work perfectly in the `Apps` section without fixing the container heights.

Reply:
Good point about the mobile views, I had not considered that.

Locally and on desktop this did fix the jitter issues for me. Is this branch deployed to explorer-beta?

Reply:
In second thought I think you should revert my commit and keep it how it is now and document this in an issue for future work.

Reply:
Opened #278 to address this later.